airbyte-integrations/bases/debezium/src/main/java/io/airbyte/integrations/debezium/internals/mongodb/MongoDbDebeziumStateUtil.java

```diff
@@ -4,6 +4,9 @@
 
 package io.airbyte.integrations.debezium.internals.mongodb;
 
+import static io.airbyte.integrations.debezium.internals.mongodb.MongoDbDebeziumConstants.OffsetState.VALUE_INCREMENT;
+import static io.airbyte.integrations.debezium.internals.mongodb.MongoDbDebeziumConstants.OffsetState.VALUE_SECONDS;
+
```

Comment:
My IDE keeps making this change automatically, and I'm not sure why it's doing it.